operatorcert/entrypoints/upload_artifacts.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+import base64
+import json
+import logging
+import os
+from typing import Any, Dict, List
+from urllib.parse import urljoin
+
+import magic
+from operatorcert.logger import setup_logger
+from operatorcert.pyxis import post_with_api_key
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> Any:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(description="Bundle dockerfile generator.")
+
+    parser.add_argument(
+        "--pyxis-url",
+        default="https://catalog.redhat.com/api/containers/",
+        help="Base URL for Pyxis container metadata API",
+    )
+    parser.add_argument(
+        "--cert-project-id", help="Certification project ID", required=True
+    )
+    parser.add_argument(
+        "--certification-hash", help="Certification bundle hash", required=True
+    )
+    parser.add_argument(
+        "--operator-package-name", help="Operator package name", required=True
+    )
+    parser.add_argument("--operator-version", help="Operator version", required=True)
+    parser.add_argument(
+        "--log-file", default="preflight.log", help="Location of preflight log file"
+    )
+    parser.add_argument(
+        "--result-file",
+        default="results.json",
+        help="Location of preflight result file",
+    )
+    parser.add_argument(
+        "--artifacts-dir",
+        default="artifacts",
+        help="Location of preflight artifacts directory",
+    )
+    parser.add_argument(
+        "--output",
+        default="output.json",
+        help="Location of output file",
+    )
+    parser.add_argument("--verbose", action="store_true", help="Verbose output")
+    return parser
+
+
+def get_artifacts(artifacts_dir: str) -> List[str]:
+    """
+    Get test artifacts files from artifacts directory
+
+    Args:
+        artifacts_dir (str): Path to artifact directory
+
+    Returns:
+        List[str]: List of artifacts files
+    """
+    artifact_paths = []
+    if not os.path.isdir(artifacts_dir):
+        LOGGER.warning(f"{artifacts_dir} is not directory")
+        return artifact_paths
+    files = os.listdir(artifacts_dir)
+    LOGGER.debug(f"Found following artifacts: {files}")
+    for artifact in files:
+        if os.path.isfile(os.path.join(artifacts_dir, artifact)):
+            artifact_paths.append(artifact)
+        else:
+            LOGGER.debug(artifact)
```

Comment:
Could you please make this log more meaningful?

Reply:
This was just a debug log for testing purposes. The log is deleted now.